common/app/views/support/WitnessCleaner.scala

```diff
@@ -1,26 +0,0 @@
-package views.support
-
-import org.jsoup.nodes.{ Element, Document }
-import scala.collection.JavaConverters._
-
-object WitnessCleaner extends HtmlCleaner {
```

Comment:
I think we have to keep the witness cleaner because there are still embeds in articles e.g. https://www.theguardian.com/australia-news/2017/sep/12/marriage-equality-rallies-around-australia-readers-photos-and-stories